blocks/identity-block/features/reset-password/default.jsx

```diff
@@ -0,0 +1,172 @@
+
+import './styles.scss';
+
+const defaultSuccessURL = '/account/login/';
+
+const ResetPassword = ({ customFields }) => {
+  const {
+    successActionURL = defaultSuccessURL,
+  } = customFields;
+
+  const { Identity, isInitialized } = useIdentity();
+  const { arcSite } = useFusionContext();
+  const { locale } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const [error, setError] = useState();
+  const [nonce, setNonce] = useState();
+  const [passwordRequirements, setPasswordRequirements] = useState({ status: 'initial' });
+  const [submitted, setSubmitted] = useState(false);
+
+  useEffect(() => {
+    // eslint doesn't handle globalThis yet, but this is appropriate
+    /* global globalThis */
+    const params = new URLSearchParams(globalThis.location.search);
+    if (params.has('nonce')) {
+      setNonce(params.get('nonce'));
+    }
+  }, []);
+
+  useEffect(() => {
+    const getConfig = async () => {
+      await Identity.getConfig()
+        .then((response) => {
+          const {
+            pwLowercase,
+            pwMinLength,
+            pwPwNumbers,
+            pwSpecialCharacters,
+            pwUppercase,
+          } = response;
+
+          setPasswordRequirements({
+            pwLowercase,
+            pwMinLength,
+            pwPwNumbers,
+            pwSpecialCharacters,
+            pwUppercase,
+            status: 'success',
+          });
+        })
+        .catch(() => setPasswordRequirements({ status: 'error' }));
+    };
+
+    if (Identity) {
+      // https://redirector.arcpublishing.com/alc/docs/swagger/?url=./arc-products/arc-identity-v1.json#/Tenant_Configuration/get
+      getConfig();
+    }
+  }, [Identity]);
+
+  const {
+    pwLowercase = 0,
+    pwMinLength = 0,
+    pwPwNumbers = 0,
+    pwSpecialCharacters = 0,
+    pwUppercase = 0,
+    status,
+  } = passwordRequirements;
+
+  const passwordErrorMessage = passwordValidationMessage({
+    defaultMessage: phrases.t('identity-block.password-requirements'),
+    options: {
+      lowercase: {
+        value: pwLowercase,
+        message: phrases.t('identity-block.password-requirements-lowercase', { requirementCount: pwLowercase }),
+      },
+      minLength: {
+        value: pwMinLength,
+        message: phrases.t('identity-block.password-requirements-characters', { requirementCount: pwMinLength }),
+      },
+      uppercase: {
+        value: pwUppercase,
+        message: phrases.t('identity-block.password-requirements-uppercase', { requirementCount: pwUppercase }),
+      },
+      numbers: {
+        value: pwPwNumbers,
+        message: phrases.t('identity-block.password-requirements-numbers', { requirementCount: pwPwNumbers }),
+      },
+      specialCharacters: {
+        value: pwSpecialCharacters,
+        message: phrases.t('identity-block.password-requirements-uppercase', { requirementCount: pwUppercase }),
+      },
+    },
+  });
+
+  if (!isInitialized || !nonce) {
```

Comment:
This is stopping anything from rendering in the PageBuilder Admin - we should allow this block to render in admin to give provide the editor with visual indication that the block is on the page

Reply:
Added the admin flag and reworked the render/tests slightly to accommodate that.  Updated the tests to use this change.  Also added storybook since it now renders with the admin change.

Reply:
Cool.